fuse.py

```diff
@@ -441,10 +445,14 @@ class as is to Operations, instead of just the fh field.
         argv = (c_char_p * len(args))(*args)
 
         fuse_ops = fuse_operations()
-        for name, prototype in fuse_operations._fields_:
+        for ent in fuse_operations._fields_:
+            name, prototype = ent[:2]
             if prototype != c_voidp and getattr(operations, name, None):
```

Comment:
Also shouldn't this branch determine which setattr(...) path you take below rather than looking at 'argtypes'?